src/skin/tooltips.cpp

```diff
@@ -723,18 +751,34 @@ void Tooltips::addStandardTooltips() {
             << tr("Next or Previous")
             << tr("Switch to either the next or previous effect.");
 
+    add("EffectSlot_metaknob")
+            << tr("Meta Knob")
+            << tr("Controls linked parameters of this effect")
+            << QString("%1: %2").arg(rightClick, resetToDefault);
+
+    add("EffectSlot_focus")
+            << tr("Effect Focus Button")
+            << QString("%1: %2").arg(leftClick, tr("Focuses this effect."))
+            << QString("%1: %2").arg(rightClick, tr("Unfocuses this effect."))
+            << tr("Refer to the web page on the Mixxx wiki for your controller for more information.");
+
     add("EffectSlot_parameter")
             << tr("Effect Parameter")
             << tr("Adjusts a parameter of the effect.")
             << QString("%1: %2").arg(rightClick, resetToDefault);
 
     add("EffectSlot_parameter_link_type")
-            << tr("Super Knob Mode")
-            << tr("Set how linked effect parameters change when turning the Super Knob.");
+            << tr("Meta Knob Link")
+            << tr("Set how this parameter is linked to the effect's Meta Knob.")
+            << tr("Empty bar: not linked")
```

Comment:
Fair enough, adding the ``state``

``Inactive state: Parameter not linked``
``Active state: Parameter moves with Meta Knob``
``Left side active state: Parameter moves with left half of Meta Knob``
``Right side active state: Parameter moves with right half of Meta Knob``
``Left and right side active state: Parameter moves across range with half of Meta Knob and back with the other half``